src/main/java/io/grpc/examples/RandomAccessSet.java

```diff
@@ -18,6 +18,10 @@ boolean isEmpty() {
   }
 
   T getRandomKey() {
```

Comment:
Can you annotate this as `@Nullable` and add a short doc saying what `null` means?